scuba/__main__.py

```diff
@@ -415,6 +416,12 @@ def get_docker_cmdline(self):
 
         args += self.options
 
+        # Precedence: (1) command-line -d, (2) alias docker_args, (3) top-level docker_args
+        if self.docker_args:
```

Comment:
In playing with this a little bit, I noticed that one cannot override `docker_args` from `.scuba.yml` to an empty list. This is because the argparse code uses `shlex.split()` on the argument string, and if given an empty string (`-d=''`), will produce an empty list, which makes this conditional False.

This is neither a showstopper, nor a problem with this PR, just something noteworthy.